pkg/fanal/artifact/vm/vm.go

```diff
@@ -15,7 +16,7 @@ import (
 	"github.com/aquasecurity/trivy/pkg/fanal/handler"
 	"github.com/aquasecurity/trivy/pkg/fanal/types"
 	"github.com/aquasecurity/trivy/pkg/fanal/walker"
-	"github.com/aquasecurity/trivy/pkg/semaphore"
+	semaphore "github.com/aquasecurity/trivy/pkg/semaphore"
```

Comment:
I'm just curious 😄
It looks like you renamed `semaphore` to `semaphore`.
Is there some reason for this?